dynet/tensor.cc

```diff
@@ -19,7 +19,7 @@ namespace dynet {
 #ifndef __CUDACC__
 
 ostream& operator<<(ostream& os, const Tensor& t) {
-  if (t.device->type == DeviceType::CPU) {
+  if ((t.device->type == DeviceType::CPU) || (t.device->type == DeviceType::ThreadPool)) {
```

Comment:
What is the reason in `model.cc` use `else if` but here use `||` together with the CPU if branch?